src/react-native-animated-charts/src/charts/linear/ChartPath.tsx

```diff
@@ -0,0 +1,380 @@
+import React, { useEffect } from 'react';
+import { Platform, View, ViewProps } from 'react-native';
+import {
+  LongPressGestureHandler,
+  LongPressGestureHandlerGestureEvent,
+  LongPressGestureHandlerProperties,
+} from 'react-native-gesture-handler';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+import Animated, {
+  cancelAnimation,
+  runOnJS,
+  runOnUI,
+  useAnimatedGestureHandler,
+  useAnimatedProps,
+  useAnimatedReaction,
+  useAnimatedStyle,
+  useSharedValue,
+  useWorkletCallback,
+  withDelay,
+  withTiming,
+} from 'react-native-reanimated';
+import * as redash from 'react-native-redash';
```

Comment:
I think we're typically trying to used named imports:
```suggestion
import { getYForX } from 'react-native-redash';
```

Reply:
Done